### PR TITLE
playSong, origVoiceOn added to context

### PR DESCRIPTION
**Pull request by cxy2015ca**
Added playSong, origVoice toggle in context, should be able to toggle the audio+visuals play pause via playSong, the vocals can be muted/unmuted with origVoice, in Preview page.
Also added the proper audio files for 月亮代表我的心, folder and file names will be the same as title_id

### CODE REVIEW

File: src/pages/Preview/containers/Preview.js [hunk 1]
```diff
@@ -2,15 +2,24 @@ import React, { useEffect, useState } from 'react';
...
 import ReactPlayer from 'react-player';
 import { getSongByTitleId } from 'utils/ktvQueries';
+import { useGlobalContext } from '../../../global/context';
 
 const Preview = ({ match }) => {
   const {
     params: { songName },
   } = match;
+  const globalContext = useGlobalContext();
+  const [karaokeState, karaokeDispatch] = globalContext.karaoke;
```
Comment (yutongliuytl) [inline src/pages/Preview/containers/Preview.js:12]: The following changes made the dependencies array error go away, adding after this line: `const { playSong } = karaokeState;`

File: src/pages/Preview/containers/Preview.js [hunk 2]
```diff
@@ -2,15 +2,24 @@ import React, { useEffect, useState } from 'react';
...
   const [songTitle, setSongTitle] = useState('');
   const [artist, setArtist] = useState('');
-  const [play, setPlay] = useState(true);
+
+  const setPlaySong = (play) => {
+    karaokeDispatch({
+      type: 'SET_PLAYSONG',
+      payload: { playSong: play },
+    });
+  };
```
Comment (yutongliuytl) [inline src/pages/Preview/containers/Preview.js]: Changing this to: ` const setPlaySong = useCallback((play) => {
      karaokeDispatch({
        type: 'SET_PLAYSONG',
        payload: { playSong: play },
      });
    },
    [karaokeDispatch],
  );`

File: src/pages/Preview/containers/Preview.js [hunk 3]
```diff
@@ -21,27 +30,46 @@ const Preview = ({ match }) => {
...
+    return () => {
+      setPlaySong(false); // stop playing when page unmount
+      // original audio should be on when unmount
+      karaokeDispatch({
+        type: 'SET_ORIGINAL_VOICE_ON',
+        payload: { origVoiceOn: true },
+      });
+      console.log(`inside preview cleanup, playSong: ${karaokeState.playSong}`);
+    };
+  }, []);
```
Comment (yutongliuytl) [inline src/pages/Preview/containers/Preview.js]: having these dependencies set to: `[karaokeDispatch, setPlaySong, playSong]`

**Review by yutongliuytl — CHANGES_REQUESTED** [review]
Tested and the audio works well damn 😎 

I did have warnings though and the quick comment changes that I added on the files worked for me, let me know if it doesn't workkk

**Review by jiaweisun-github — APPROVED** [review]

**Comment by cxy2015ca** [comment]
@yutongliuytl changed

**Review by xinruili07 — APPROVED** [review]

**Review by yutongliuytl — APPROVED** [review]
👍

**Pull request merged**